docs/materialeditorwidget.rst

```diff
@@ -1,38 +1,60 @@
 
    Material editor widget.
 
 Material list
 -------------
 
-The material list displays all currently defined materials; it will always contain the default material if no other materials are defined.
-The two buttons at the top of the window allow you to create or delete materials.
+Along the top of the material editor window are two buttons; create and delete.  You can create a new material, or delete an existing material using these buttons.  
 
-Settings editor
----------------
+Below these buttons is the list of currently defined materials.
+These will contain the default materials, as well as any defined in any comfile that has been run.
 
-Below the material list are controls that allow you to set some general properties of the selected material.
-Ambient Color, Diffuse Color, Emission Color, and Specular Color. 
-You can also adjust the alpha and shininess of the material by entering a number between 0 and 1 or dragging the sliders.
+You can rename the material by double click it.
 
-Preview panel
--------------
+.. figure:: _images/material-editor-rename.png
+   :alt: Material editor rename material.
+   :width: 50%
 
-This panel displays a sphere, shaded with the selected material.
+   Rename material.
+
+Controls
+--------
+
+Below the material list is a panel containing four colour select buttons. These control the ambient, diffuse, emitted and specular colours.
 
-API
----
+* Ambient - The ambient colour is the "unlit" colour, or the colour of parts of the object that are in shadow.
+* Diffuse - This is the overall colour of the material, the colour that the lit parts of the object will appear.
+* Emitted - The emitted colour is the "glow" of a material; this colour will appear in both the lit and unlit parts of the material.
+* Specular - This is the colour of the shine that appears on the material.  This shine appears as a glossy highlight.
 
-.. autoclass:: opencmiss.zincwidgets.materialeditorwidget.MaterialEditorWidget
-   :members:
+.. figure:: _images/material-editor-select-color.png
+   :alt: Material editor select colour.
+   :width: 50%
 
-.. autoclass:: opencmiss.zincwidgets.materialeditorwidget.MaterialModel
-   :members:
\ No newline at end of file
+   Select colour.
+
+When you click the Select Colour button, a colour dialog will pop up, and you can select a colour in several ways. The background of these four buttons indicates the colour you choose.
+
+Below the colour editors is the surface editor.  This panel allows you to set the alpha, shininess, and texture properties of the surface of the material being edited.  
+The alpha value sets the transparency of the material.  The shininess sets the "tightness" or size of the specular highlights of a material; generally the higher the shininess, the smaller and harder-edged the highlights.  
+Higher shininess makes a material look glossier.  
+The surface editor also allows you to assign a texture to the material surface - this option is unavailable for now.
```

Comment:
Below the colour editors are further attributes controlling the rendering:

* *Alpha* sets the opacity, from 0.0 (transparent) to 1.0 (opaque) with corresponding variable translucency in between. Use of alpha < 1.0 may require use of Sceneviewer Transparency modes `slow` or `order independent` for best results.
* *Shininess* sets the "tightness" or size of the specular highlights of a material, a value from 0.0 to 1.0. Generally, the higher the shininess, the smaller and harder-edged are the highlights. Use requires a non-black specular colour.
* *Texture* specifies up to 4 image fields from a region to be attached to the material for texture mapping and other special effects. Note at this time facilities for loading images and shaders which use these have not been exposed.